sairameshv/machine-config-operator
# Kubelet config

**- What I did**
Added code to create/update KubeletConfig CR whenever there is a change in the Node CR
**- How to verify it**
Modify the Node CR and observe the logs in the Machine Config Controller pod corresponding to the create/update of the KubeletConfig CR
**- Description for the changelog**
Instead of modifying the MachineConfig CR, this implementation leverages the KubeletConfig CR whenever there is a change in the Node CR.
